ui/pages/swaps/fee-card/fee-card.js

```diff
@@ -109,7 +109,6 @@ export default function FeeCard({
                     }
                     containerClassName="fee-card__info-tooltip-content-container"
                     wrapperClassName="fee-card__row-label fee-card__info-tooltip-container"
-                    wide
                   />
                 </>
               }
```

Comment:
Removing the once instance of the `wide` prop

![Screen Shot 2022-06-20 at 1 37 59 PM](https://user-images.githubusercontent.com/8112138/174675441-fa52afdc-1e81-4b24-8555-314a079ef9b4.png)

Looks fine with same width as all the other InfoTooltips

<img width="365" alt="Screen Shot 2022-06-20 at 1 23 15 PM" src="https://user-images.githubusercontent.com/8112138/174675458-ee5cd166-3c8b-467f-af37-63ade94f794f.png">